docs/markdown/material.md

```diff
@@ -0,0 +1,28 @@
+### Material Design
+
+MyUW will soon make a transition from Bootstrap to Angular Material, as an implementation of Google's [Material Design](https://www.google.com/design/spec/material-design/introduction.html). The main reasons for doing this are:
+
++ To allow UX Designers to focus on UW-specific designs and problems rather than reinventing the wheel for simple components
++ To have a structured, unified design philosophy that works well together. The hope is that Material 'just works' for us.
++ Material gives several more complicated UI elements and features for free - animations, toasts, trainstops, etc.
+
+This page will serve as a resource for Material documentation, and how MyUW uses it.
+
+#### Material Components in MyUW
+
++ **[Buttons](https://material.angularjs.org/latest/demo/button)**: Used throughout MyUW.
+```
+<md-button class="md-raised">Button</md-button>
+<md-button class="md-raised md-primary">Primary</md-button>
+<md-button ng-disabled="true" class="md-raised md-primary">Disabled</md-button>
+<md-button class="md-raised md-warn">Warn</md-button>
+```
++ **[Fab Buttons](https://material.angularjs.org/latest/demo/button)**: Used for links inside of widgets. 
+```
+<md-button class="md-fab md-mini md-primary" aria-label="Use Android">
+  <md-icon md-svg-src="img/icons/android.svg" style="color: greenyellow;"></md-icon>
+</md-button>
```

Comment:
Looking at the material docs, you are actually not supposed to use the FAB buttons in this way. Checkout https://www.google.com/design/spec/components/buttons-floating-action-button.html for details. Not sure what we will do about the circle buttons in widgets, maybe something like fab, but makes one rethink the circle button approach.